### ISSUE TITLE
Add Front packages

### ISSUE THREAD

**Issue by theofidry** [in6pio/Incipio#26]
Add [nodejs](https://nodejs.org/):

```
curl -sL https://deb.nodesource.com/setup | sudo bash -
sudo apt-get install nodejs
```

or http://konradpodgorski.com/blog/2014/06/22/how-to-install-node-js/

Install gulp globally for more comfort:

```
npm install gulp
```

Install [SASS](http://sass-lang.com/); Usefull at least for the interactive shell.

**Comment by theofidry** [comment]
Goes with #33 so is reported to the next milestone.

**Comment by theofidry** [comment]
Closed via a6096b90bd1c7847b781ef7ed3101a7e707ef5e9
